checks/fuzzing.go

```diff
@@ -45,5 +46,13 @@ func Fuzzing(c *checker.CheckRequest) checker.CheckResult {
 		c.RawResults.FuzzingResults = rawData
 	}
 
-	return evaluation.Fuzzing(CheckFuzzing, c.Dlogger, &rawData)
+	// Evaluate the probes.
+	findings, err := evaluateProbes(c, CheckFuzzing, probes.Fuzzing)
+	if err != nil {
```

Comment:
this has the same issue where we panic if `c.RawResults == nil`. And we have the data in `rawData`, so we are losing some functionality here.